includes/Classifai/Providers/OpenAI/TextToSpeech.php

```diff
@@ -0,0 +1,323 @@
+					wp_kses(
+						/* translators: %1$s is replaced with the OpenAI sign up URL */
+						__( 'Don\'t have an OpenAI account yet? <a title="Sign up for an OpenAI account" href="%1$s">Sign up for one</a> in order to get your API key.', 'classifai' ),
+						[
+							'a' => [
+								'href'  => [],
+								'title' => [],
+							],
+						]
+					),
+					esc_url( 'https://platform.openai.com/signup' )
+				),
+			]
+		);
+
+		add_settings_field(
+			static::ID . '_tts_model',
+			esc_html__( 'TTS model', 'classifai' ),
+			[ $this->feature_instance, 'render_select' ],
+			$this->feature_instance->get_option_name(),
+			$this->feature_instance->get_option_name() . '_section',
+			[
+				'option_index'  => static::ID,
+				'label_for'     => 'tts_model',
+				'options'       => [
+					'tts-1'    => __( 'Text-to-speech 1 (Optimized for speed)', 'classifai' ),
+					'tts-1-hd' => __( 'Text-to-speech 1 HD (Optimized for quality)', 'classifai' ),
+				],
+				'default_value' => $settings['tts_model'],
+				'description'   => __( 'Select a model depending on your requirements.', 'classifai' ),
+				'class'         => 'classifai-provider-field hidden provider-scope-' . static::ID,
+			]
+		);
+
+		add_settings_field(
+			static::ID . '_voice',
+			esc_html__( 'Voice', 'classifai' ),
+			[ $this->feature_instance, 'render_select' ],
+			$this->feature_instance->get_option_name(),
+			$this->feature_instance->get_option_name() . '_section',
+			[
+				'option_index'  => static::ID,
+				'label_for'     => 'voice',
+				'options'       => [
+					'alloy'   => __( 'Alloy (male)', 'classifai' ),
+					'echo'    => __( 'Echo (male)', 'classifai' ),
+					'fable'   => __( 'Fable (male)', 'classifai' ),
+					'onyx'    => __( 'Onyx (male)', 'classifai' ),
+					'nova'    => __( 'Nova (female)', 'classifai' ),
+					'shimmer' => __( 'Shimmer (female)', 'classifai' ),
+				],
+				'default_value' => $settings['voice'],
+				'description'   => __( 'Select the desired speech voice.', 'classifai' ),
+				'class'         => 'classifai-provider-field hidden provider-scope-' . static::ID,
+			]
+		);
+
+		add_settings_field(
+			static::ID . '_format',
+			esc_html__( 'Audio format', 'classifai' ),
+			[ $this->feature_instance, 'render_select' ],
+			$this->feature_instance->get_option_name(),
+			$this->feature_instance->get_option_name() . '_section',
+			[
+				'option_index'  => static::ID,
+				'label_for'     => 'format',
+				'options'       => [
+					'mp3'  => __( '.mp3', 'classifai' ),
+					'opus' => __( '.opus', 'classifai' ),
+					'aac'  => __( '.aac', 'classifai' ),
+					'flac' => __( '.flac', 'classifai' ),
+					'wav'  => __( '.wav', 'classifai' ),
+					'pcm'  => __( '.pcm', 'classifai' ),
+				],
+				'default_value' => $settings['format'],
+				'description'   => __( 'Select the desired audio format.', 'classifai' ),
+				'class'         => 'classifai-provider-field hidden provider-scope-' . static::ID,
+			]
+		);
```

Comment:
I think maybe we just get rid of this and hardcode to the `.mp3` format. I get that these are additional formats the API supports but not sure WordPress even supports all of these formats natively and for the sake of limiting options, I'd just remove this

Reply:
I've limited the formats to `wav` and `mp3`.